### PR TITLE
Update the F# Guide page and some ms.date links

### PR DESCRIPTION
**Pull request by cartermp**
Centered around a better getting started experience for F#. This is based on looking at how other languages document content for newcomers and other learnings in this space. There will be more PRs to address further content.

### CODE REVIEW

File: docs/fsharp/index.md [hunk 1]
```diff
@@ -14,45 +14,71 @@ ms.assetid: ea27fb37-dad1-4bd4-a3cc-4f5c70767ae9
...
-* [Introduction to Programming with F# series on YouTube](https://www.youtube.com/watch?v=Teak30_pXHk&list=PLEoMzSkcN8oNiJ67Hd7oRGgD1d4YBxYGC)
-* [Introduction to F# series on FSharpTV](https://fsharp.tv/courses/fsharp-programming-intro/)
+[What's New in Visual Studio 2017 for F# For Developers](https://www.linkedin.com/learning/what-s-new-in-visual-studio-2017-for-f-sharp-for-developers) is a video course which shows some of the newer features for F# in Visual Studio 2017.
 
-## Further Resources
+## Other useful resources
+
+The [F# Snippets Website](http://www.fssnip.net) contains a massive set of code snippets showing how to do just about anything in F#, ranging from absolute beginner to highly advanced snippets.
+
+The [F# Software Foundation Slack]()
```
Comment (cartermp) [inline docs/fsharp/index.md]: whoops

File: docs/fsharp/index.md [hunk 2]
```diff
@@ -14,45 +14,71 @@ ms.assetid: ea27fb37-dad1-4bd4-a3cc-4f5c70767ae9
...
+    args // Use F# pipe operators because they're awesome!
+    |> Array.map getGreeting // Turn each name into a friendly greeting!
+    |> Array.iter printfn // Print them!
 
-If you know your way around F#, or want to learn more about a specific language construct, see the [Language Reference](language-reference/index.md).  It's a thorough guide of all F# language capabilities.
+    0 // 0 return code
+```
 
-Additionally, the [F# Core Library Reference](https://msdn.microsoft.com/visualfsharpdocs/conceptual/fsharp-core-library-reference) is a great resource for learning about FSharp.Core, the core library which is a part of F#.
+F# is about productivity at its heart. The tooling support for F# is ubiquitous and full of advanced features. See below to learn more.
```
Comment (mairaw) [inline docs/fsharp/index.md]: are you working on VS Code? If so, can you install and use this extension:
https://review.docs.microsoft.com/en-us/help/contribute/contribute-get-started-setup-acrolinx-vscode?branch=master

It will capture some of the common style guide rules like don't use below.

File: docs/fsharp/index.md [hunk 3]
```diff
@@ -14,45 +14,71 @@ ms.assetid: ea27fb37-dad1-4bd4-a3cc-4f5c70767ae9
...
+[Tour of F#](tour.md) gives an overview of major language features with lots of code samples. Go here if you're interested in language concepts.
 
-The mission of the F# Software Foundation is to promote, protect, and advance the F# programming language, and to support and facilitate the growth of a diverse and international community of F# programmers.
+[Get started with F# in Visual Studio](get-started/get-started-visual-studio.md) if you're on Windows and want a the full Visual Studio IDE experience.
+
+[Get started with F# in Visual Studio for Mac](get-started/get-started-visual-studio-for-mac.md) if you're on macOS.
+
+[Get Started with F# in Visual Studio Code](get-started/get-started-vscode.md) if you want a lightweight and feature-packed IDE experience.
+
+[Get started with F# with the .NET Core CLI](get-started/get-started-command-line.md) if you just want a great set of command line tools.
```
Comment (mairaw) [inline docs/fsharp/index.md]: command line -> command-line

File: docs/fsharp/index.md [hunk 4]
```diff
@@ -14,45 +14,71 @@ ms.assetid: ea27fb37-dad1-4bd4-a3cc-4f5c70767ae9
...
+## Additional guides
 
-## Documentation
+[F# for Fun and Profit](https://swlaschin.gitbooks.io/fsharpforfunandprofit/content/) is a comprehensive and very detailed book on learning F#. Its contents and author are beloved by the F# community.
 
-* [Tutorials](tutorials/getting-started/index.md)
-* [Functions as First-Class Values](introduction-to-functional-programming/functions-as-first-class-values.md)<!--[Introduction to Functional Programming](introduction-to-functional-programming/index.md)-->
-* [Language Reference](language-reference/index.md)
-* [F# Core Library Reference](https://msdn.microsoft.com/visualfsharpdocs/conceptual/fsharp-core-library-reference)
+[F# Programming Wikibook](https://en.wikibooks.org/wiki/F_Sharp_Programming) is a wonderful wikibook on learning F#. It is also a product of the F# community.
```
Comment (mairaw) [inline docs/fsharp/index.md]: I'd avoid adjectives like wonderful to describe things.

File: docs/fsharp/get-started/get-started-vscode.md [hunk 5]
```diff
@@ -1,18 +1,18 @@
 ---
-title: Getting Started with F# in Visual Studio Code with Ionide
+title: Getting Started with F# in Visual Studio Code
 description: Learn how to use F# with Visual Studio Code and the Ionide plugin suite.
 keywords: visual f#, f#, functional programming, .NET, Visual Studio Code, vscode, Ionide
 author: cartermp
 ms.author: phcart
-ms.date: 09/28/2016
+ms.date: 02/28/2018
```
[the remark this answers is not in the document]
Comment (cartermp) [reply docs/fsharp/get-started/get-started-vscode.md]: Re: 4.1 - I'm actually just going to remove that section. It's confusing, and the standalone install of F# atop Windows is something very few people do.

Re: Mono - still untrue, unfortunately. The .NET Core-only Ionide experience is still experimental.

**Comment by BillWagner** [comment]
@cartermp This branch has merge conflicts. Do you want to rebase and then I'll review?

**Comment by cartermp** [comment]
@BillWagner done!

**Comment by mairaw** [comment]
I'll take a look but build report is showing a broken link already:
image

File: docs/fsharp/get-started/get-started-vscode.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (mairaw) [reply docs/fsharp/get-started/get-started-vscode.md]: Sounds good @cartermp!

File: docs/fsharp/get-started/get-started-vscode.md [hunk 6]
```diff
@@ -1,65 +1,28 @@
...
-On 32-bit Windows, the compiler tools are located here:
-
-```
-C:\Program Files\Microsoft SDKs\F#\4.0\Framework\v4.0\fsc.exe
-C:\Program Files\Microsoft SDKs\F#\4.0\Framework\v4.0\fsi.exe
-C:\Program Files\Microsoft SDKs\F#\4.0\Framework\v4.0\fsiAnyCpu.exe
-```
-
-Ionide automatically detects the compiler and tools, but if it doesn't for some reason (for example, the Visual F# Tools were installed to a different directory), you can manually add the containing folder (`...\Microsoft SDKs\F#\4.0`) to your PATH.
+If you're on Windows, you must [install Visual Studio with F# support](get-started-visual-studio.md#installing-f).  This will install all the necessary components to write, compile, and execute F# code.
```
Comment (mairaw) [inline docs/fsharp/get-started/get-started-vscode.md]: tabs might come handy in these subsections:
https://review.docs.microsoft.com/en-us/new-hope/resources/tabbed-conceptual?branch=master&tabs=linux%2Cazure-cli#os 

Do you need to add more details here? People might be surprised that in order for them to Get started with F# on VS Code, they also need to install the full VS as well.

will install -> installs

File: docs/fsharp/get-started/get-started-vscode.md [hunk 7]
```diff
@@ -1,65 +1,28 @@
...
+# Get Started with F# in Visual Studio Code
 
 You can write F# in [Visual Studio Code](https://code.visualstudio.com) with the [Ionide plugin](https://marketplace.visualstudio.com/items?itemName=Ionide.Ionide-fsharp), to get a great cross-platform, lightweight IDE experience with IntelliSense and basic code refactorings.  Visit [Ionide.io](https://ionide.io) to learn more about the plugin suite.
 
 ## Prerequisites
 
-F# 4.0 or higher must be installed on your machine to use Ionide.
-
-You must also have [git installed](https://git-scm.com/download) and available on your PATH to make use of project templates in Ionide.  You can verify that it is installed correctly by typing `git` at a command prompt.and pressing **Enter**.
+You must have [git installed](https://git-scm.com/download) and available on your PATH to make use of project templates in Ionide.  You can verify that it is installed correctly by typing `git` at a command prompt and pressing **Enter**.
```
Comment (mairaw) [inline docs/fsharp/get-started/get-started-vscode.md]: This seems to be missing a couple of prereqs like Visual Studio Code and the Ionide plug-in.

File: docs/fsharp/get-started/get-started-vscode.md [hunk 8]
```diff
@@ -1,65 +1,28 @@
...
 ms.prod: .net
 ms.technology: devlang-fsharp
 ms.devlang: fsharp
 ms.assetid: 49775139-082e-442f-b5a2-dd402399b5d2
 ---
 
-# Getting Started with F# in Visual Studio Code with Ionide
+# Get Started with F# in Visual Studio Code
 
 You can write F# in [Visual Studio Code](https://code.visualstudio.com) with the [Ionide plugin](https://marketplace.visualstudio.com/items?itemName=Ionide.Ionide-fsharp), to get a great cross-platform, lightweight IDE experience with IntelliSense and basic code refactorings.  Visit [Ionide.io](https://ionide.io) to learn more about the plugin suite.
```
Comment (mairaw) [inline docs/fsharp/get-started/get-started-vscode.md]: not related to your changes but remove the extra space before Visit

File: docs/fsharp/index.md [hunk 9]
```diff
@@ -1,10 +1,9 @@
 ---
 title: F# Guide
-description: Learn about F#, a functional programming language that runs on .NET.
-keywords: .NET, .NET Core
+description: Learn F#, a functional programming language that runs on .NET.
```
Comment (mairaw) [inline docs/fsharp/index.md]: Try to expand this a bit more. The length should be between 115 and 145 characters. Maybe tell folks what they'll find in the Guide?

File: docs/fsharp/index.md [hunk 10]
```diff
@@ -14,45 +13,69 @@ ms.assetid: ea27fb37-dad1-4bd4-a3cc-4f5c70767ae9
...
+[F# for Fun and Profit](https://swlaschin.gitbooks.io/fsharpforfunandprofit/content/) is a comprehensive and very detailed book on learning F#. Its contents and author are beloved by the F# community.
 
-* [Tutorials](tutorials/getting-started/index.md)
-* [Functions as First-Class Values](introduction-to-functional-programming/functions-as-first-class-values.md)<!--[Introduction to Functional Programming](introduction-to-functional-programming/index.md)-->
-* [Language Reference](language-reference/index.md)
-* [F# Core Library Reference](https://msdn.microsoft.com/visualfsharpdocs/conceptual/fsharp-core-library-reference)
+[F# Programming Wikibook](https://en.wikibooks.org/wiki/F_Sharp_Programming) is a wikibook about learning F#. It is also a product of the F# community.
 
-## Online Reading Resources
+## Learn F# by video
```
Comment (mairaw) [inline docs/fsharp/index.md]: reads a bit funny like that. Learn through videos or with videos perhaps?

File: docs/fsharp/index.md [hunk 11]
```diff
@@ -14,45 +13,69 @@ ms.assetid: ea27fb37-dad1-4bd4-a3cc-4f5c70767ae9
...
+```
 
-Additionally, the [F# Core Library Reference](https://msdn.microsoft.com/visualfsharpdocs/conceptual/fsharp-core-library-reference) is a great resource for learning about FSharp.Core, the core library which is a part of F#.
+F# is about productivity at its heart. The tooling support for F# is ubiquitous and full of advanced features.
 
-## The F# Software Foundation
+## Learning F# #
 
-Although Microsoft is the primary developer of the F# language and its tooling, F# is also backed by an independent foundation, the F# Software Foundation (FSSF).
+[Tour of F#](tour.md) gives an overview of major language features with lots of code samples. Go here if you're interested in getting an overview of the language.
```
Comment (mairaw) [inline docs/fsharp/index.md]: Acrolinx result: Go here -> do not use

File: docs/fsharp/index.md [hunk 12]
```diff
@@ -14,45 +13,69 @@ ms.assetid: ea27fb37-dad1-4bd4-a3cc-4f5c70767ae9
...
 
-## Online Reading Resources
+## Learn F# by video
 
-* [F# for Fun and Profit Gitbook](https://swlaschin.gitbooks.io/fsharpforfunandprofit/content/) 
-* [F# Programming Wikibook](https://en.wikibooks.org/wiki/F_Sharp_Programming)
+[F# tutorial on YouTube](https://www.youtube.com/watch?v=c7eNDJN758U) is a great introduction to F# using Visual Studio, showing lots of great examples over the course of 1.5 hours.
 
-## Video Learning Resources
+[Introduction to Programming with F#](https://www.youtube.com/watch?v=Teak30_pXHk&list=PLEoMzSkcN8oNiJ67Hd7oRGgD1d4YBxYGC) is a great video series which uses Visual Studio Code as the main editor. The video series starts from nothing and ends with building a text-based RPG video game.
```
Comment (mairaw) [inline docs/fsharp/index.md]: Acrolinx result: video series which -> video series that

File: docs/fsharp/index.md [hunk 13]
```diff
@@ -14,45 +13,69 @@ ms.assetid: ea27fb37-dad1-4bd4-a3cc-4f5c70767ae9
...
 
-* [Introduction to Programming with F# series on YouTube](https://www.youtube.com/watch?v=Teak30_pXHk&list=PLEoMzSkcN8oNiJ67Hd7oRGgD1d4YBxYGC)
-* [Introduction to F# series on FSharpTV](https://fsharp.tv/courses/fsharp-programming-intro/)
+[What's New in Visual Studio 2017 for F# For Developers](https://www.linkedin.com/learning/what-s-new-in-visual-studio-2017-for-f-sharp-for-developers) is a video course which shows some of the newer features for F# in Visual Studio 2017.
```
Comment (mairaw) [inline docs/fsharp/index.md]: Acrolinx result: course which -> course that

File: docs/fsharp/index.md [hunk 14]
```diff
@@ -14,45 +13,69 @@ ms.assetid: ea27fb37-dad1-4bd4-a3cc-4f5c70767ae9
...
+
+Although Microsoft is the primary developer of the F# language and its tools in Visual Studio, F# is also backed by an independent foundation, the F# Software Foundation (FSSF).
+
+The mission of the F# Software Foundation is to promote, protect, and advance the F# programming language, and to support and facilitate the growth of a diverse and international community of F# programmers.
 
-* [F# Learning Resources on fsharp.org](http://fsharp.org/learn.html)
-* [F# Snippets Website](http://www.fssnip.net)
-* [F# Software Foundation](http://fsharp.org)
\ No newline at end of file
+To learn more and get involved, check out [fsharp.org](http://fsharp.org). It's 100% free to join, and the network of F# developers in the foundation is something you won't want to miss out on!
```
Comment (mairaw) [inline docs/fsharp/index.md]: Acrolinx result: won't want -> don't want

**Review by mairaw — CHANGES_REQUESTED** [review]
Changes look good overall. Highlighted a couple of suggestions from Acrolinx too.

File: docs/fsharp/get-started/get-started-vscode.md (hunk 8 above)
Comment (cartermp) [reply docs/fsharp/get-started/get-started-vscode.md]: Actually, my next action after this PR is to update this article, so I'll take care of the space-y problems then.

File: docs/fsharp/get-started/get-started-vscode.md (hunk 7 above)
Comment (cartermp) [reply docs/fsharp/get-started/get-started-vscode.md]: Mentioned right below. These are prerequisities to actually get F# on your machine in a way that Ionide can use it.

File: docs/fsharp/get-started/get-started-vscode.md (hunk 6 above)
Comment (cartermp) [reply docs/fsharp/get-started/get-started-vscode.md]: Added/addressed

**Comment by cartermp** [comment]
@mairaw and @BillWagner any additional feedback?

File: docs/fsharp/get-started/get-started-vscode.md [hunk 15]
```diff
@@ -1,83 +1,48 @@
...
+On Linux, Ionide also uses [Mono](http://www.mono-project.com).  If you're on Debian or Ubuntu, you can use the following:
 
 ```
 sudo apt-get update
 sudo apt-get install mono-complete fsharp
 ```
 
+### [Windows](#tab/windows)
+
+If you're on Windows, you must [install Visual Studio with F# support](get-started-visual-studio.md#installing-f).  This installs all the necessary components to write, compile, and execute F# code.
```
Comment (svick) [inline docs/fsharp/get-started/get-started-vscode.md]: Since installing VS is not the only option, I wouldn't use the word "must" here, even if you're not going to mention the other option.

Also, I find it weird that to use F# in one editor (VS Code), the easiest option is to first install another editor (VS). But I don't think you can fix that by changing what the docs say. 😄

File: docs/fsharp/get-started/get-started-vscode.md [hunk 16]
```diff
@@ -117,7 +82,7 @@ Select "F#: New Project" by hitting **Enter**, which will take you to this step:
 
 ![](media/getting-started-vscode/vscode-proj-type.png)
 
-This will select a template for a specific type of project.  There are quite a few options here, such as an [FsLab](https://fslab.org) template for Data Science or [Suave](https://suave.io) template for Web Programming.  This article uses the `classlib` template, so highlight that and hit **Enter**.  You will then reach the following step:
+This will select a template for a specific type of project.  There are quite a few options here, such as an [FsLab](http://fslab.org) template for Data Science or [Suave](https://suave.io) template for Web Programming.  This article uses the `classlib` template, so highlight that and hit **Enter**.  You will then reach the following step:
```
Comment (svick) [inline docs/fsharp/get-started/get-started-vscode.md]: Why did you change the URL to HTTP here, when http://fslab.org redirects to https://fslab.org? I think it makes sense to use HTTPS URLs by default for sites where HTTPS works.

File: docs/fsharp/get-started/get-started-vscode.md (hunk 15 above)
Comment (cartermp) [reply docs/fsharp/get-started/get-started-vscode.md]: Until we complete F# support for FSI and Type Providers on .NET Core, Ionide must take a dependency on either Mono or .NET Framework and their build tools. This means that you need to manually install the .NET Framework (win7 or higher), F# Compiler SDK,  MSBuild, and optionally set your PATHs appropriately if Ionide cannot detect them (reported before so it should be mentioned).

Having watched people struggle mightily with this in workshops, even with this doc, I've had them "just install VS" to make it easier and they were appreciative 😢. The removal here is based on this.

And actually, now that you've hit on this, this is not finished yet, since the .NET Core SDK is required for Ionide as well. I'll need to update that.

File: docs/fsharp/get-started/get-started-vscode.md (hunk 16 above)
Comment (cartermp) [reply docs/fsharp/get-started/get-started-vscode.md]: Ah, this is fun. So I cloned this repo before @mairaw's PR to change all links to https, so it now registers as a change.

File: docs/fsharp/get-started/get-started-vscode.md [hunk 17]
```diff
@@ -1,82 +1,53 @@
...
 ms.technology: devlang-fsharp
 ms.devlang: fsharp
 ms.assetid: 49775139-082e-442f-b5a2-dd402399b5d2
 ---
 
-# Getting Started with F# in Visual Studio Code with Ionide
+# Get Started with F# in Visual Studio Code
 
-You can write F# in [Visual Studio Code](https://code.visualstudio.com) with the [Ionide plugin](https://marketplace.visualstudio.com/items?itemName=Ionide.Ionide-fsharp), to get a great cross-platform, lightweight IDE experience with IntelliSense and basic code refactorings.  Visit [Ionide.io](https://ionide.io) to learn more about the plugin suite.
+You can write F# in [Visual Studio Code](https://code.visualstudio.com) with the [Ionide plugin](https://marketplace.visualstudio.com/items?itemName=Ionide.Ionide-fsharp), to get a great cross-platform, lightweight IDE experience with IntelliSense and basic code refactorings.  Visit [Ionide.io](http://ionide.io) to learn more about the plugin suite.
```
Comment (BillWagner) [inline docs/fsharp/get-started/get-started-vscode.md]: I don't think we want http instead of https. however, ionide.io doesn't support https. @mairaw what do you recommend?

File: docs/fsharp/get-started/get-started-vscode.md [hunk 18]
```diff
@@ -1,82 +1,53 @@
...
 
 ## Prerequisites
 
-F# 4.0 or higher must be installed on your machine to use Ionide.
+You must have [git installed](https://git-scm.com/download) and available on your PATH to make use of project templates in Ionide.  You can verify that it is installed correctly by typing `git` at a command prompt and pressing **Enter**.
```
Comment (BillWagner) [inline docs/fsharp/get-started/get-started-vscode.md]: Nit / suggestion: I usually say to type `git --version` to get version information as well.

File: docs/fsharp/get-started/get-started-vscode.md [hunk 19]
```diff
@@ -1,82 +1,53 @@
...
 
 ```
-C:\Program Files\Microsoft SDKs\F#\4.0\Framework\v4.0\fsc.exe
-C:\Program Files\Microsoft SDKs\F#\4.0\Framework\v4.0\fsi.exe
-C:\Program Files\Microsoft SDKs\F#\4.0\Framework\v4.0\fsiAnyCpu.exe
+brew install mono
 ```
 
-Ionide automatically detects the compiler and tools, but if it doesn't for some reason (for example, the Visual F# Tools were installed to a different directory), you can manually add the containing folder (`...\Microsoft SDKs\F#\4.0`) to your PATH.
+You must also install the [.NET Core SDK](https://www.microsoft.com/net/download/macos).
```
Comment (BillWagner) [inline docs/fsharp/get-started/get-started-vscode.md]: nit / suggestion: you can omit '/macos' and it will auto detect the user's OS.

File: docs/fsharp/get-started/get-started-vscode.md [hunk 20]
```diff
@@ -1,82 +1,53 @@
...
+On Linux, Ionide also uses [Mono](http://www.mono-project.com).  If you're on Debian or Ubuntu, you can use the following:
 
 ```
-brew install mono
+sudo apt-get update
+sudo apt-get install mono-complete fsharp
 ```
 
-### Linux
+You must also install the [.NET Core SDK](https://www.microsoft.com/net/download/linux).
```
Comment (BillWagner) [inline docs/fsharp/get-started/get-started-vscode.md]: same comment as the mac installation.

File: docs/fsharp/get-started/get-started-vscode.md [hunk 21]
```diff
@@ -1,82 +1,53 @@
...
-On Linux, Ionide also uses [Mono](https://www.mono-project.com).  If you're on Debian or Ubuntu, you can use the following:
+### [Windows](#tab/windows)
 
-```
-sudo apt-get update
-sudo apt-get install mono-complete fsharp
-```
+If you're on Windows, you must [install Visual Studio with F# support](get-started-visual-studio.md#installing-f).  This installs all the necessary components to write, compile, and execute F# code.
+
+You must also install the [.NET Core SDK](https://www.microsoft.com/net/download/windows/).
```
Comment (BillWagner) [inline docs/fsharp/get-started/get-started-vscode.md]: again, the page detects the OS from the browser.

File: docs/fsharp/index.md [hunk 22]
```diff
@@ -14,45 +13,69 @@ ms.assetid: ea27fb37-dad1-4bd4-a3cc-4f5c70767ae9
...
+```
 
-Additionally, the [F# Core Library Reference](https://msdn.microsoft.com/visualfsharpdocs/conceptual/fsharp-core-library-reference) is a great resource for learning about FSharp.Core, the core library which is a part of F#.
+F# is about productivity at its heart. The tooling support for F# is ubiquitous and full of advanced features.
 
-## The F# Software Foundation
+## Learning F# #
 
-Although Microsoft is the primary developer of the F# language and its tooling, F# is also backed by an independent foundation, the F# Software Foundation (FSSF).
+[Tour of F#](tour.md) gives an overview of major language features with lots of code samples. This is recommended if you are new to F# and want to get a feel for how the language works.
```
Comment (BillWagner) [inline docs/fsharp/index.md]: Would the tour also be useful for new developers? If not, is there a new programmers introduction to F#?

File: docs/fsharp/index.md [hunk 23]
```diff
@@ -14,45 +13,69 @@ ms.assetid: ea27fb37-dad1-4bd4-a3cc-4f5c70767ae9
...
+## Additional guides
 
-## Documentation
+[F# for Fun and Profit](https://swlaschin.gitbooks.io/fsharpforfunandprofit/content/) is a comprehensive and very detailed book on learning F#. Its contents and author are beloved by the F# community.
 
-* [Tutorials](tutorials/getting-started/index.md)
-* [Functions as First-Class Values](introduction-to-functional-programming/functions-as-first-class-values.md)<!--[Introduction to Functional Programming](introduction-to-functional-programming/index.md)-->
-* [Language Reference](language-reference/index.md)
-* [F# Core Library Reference](https://msdn.microsoft.com/visualfsharpdocs/conceptual/fsharp-core-library-reference)
+[F# Programming Wikibook](https://en.wikibooks.org/wiki/F_Sharp_Programming) is a wikibook about learning F#. It is also a product of the F# community.
```
Comment (BillWagner) [inline docs/fsharp/index.md]: Who would be the target audience for the wikibook? That would be important for this audience.

Reading on, this comment applies to all the resources you mention: If you know the target audience, that would be useful to know.

File: docs/fsharp/index.md [hunk 24]
```diff
@@ -14,45 +13,69 @@ ms.assetid: ea27fb37-dad1-4bd4-a3cc-4f5c70767ae9
...
+
+Although Microsoft is the primary developer of the F# language and its tools in Visual Studio, F# is also backed by an independent foundation, the F# Software Foundation (FSSF).
+
+The mission of the F# Software Foundation is to promote, protect, and advance the F# programming language, and to support and facilitate the growth of a diverse and international community of F# programmers.
 
-* [F# Learning Resources on fsharp.org](http://fsharp.org/learn.html)
-* [F# Snippets Website](http://www.fssnip.net)
-* [F# Software Foundation](http://fsharp.org)
\ No newline at end of file
+To learn more and get involved, check out [fsharp.org](http://fsharp.org). It's 100% free to join, and the network of F# developers in the foundation is something you don't want to miss out on!
```
Comment (BillWagner) [inline docs/fsharp/index.md]: I'd remove this, and move the link to the text "..., the F# Software Foundation (FSSF)"

**Review by BillWagner — APPROVED** [review]
Overall, this looks great. I'm approving this to merge. I have a few suggestions that you can adopt as you see fit.

File: docs/fsharp/get-started/get-started-vscode.md (hunk 17 above)
Comment (mairaw) [reply docs/fsharp/get-started/get-started-vscode.md]: I tried to keep only the links that worked with https but I guess I made a mistake here. I'd keep as http on this one.

File: docs/fsharp/get-started/get-started-vscode.md [hunk 25]
```diff
@@ -1,82 +1,53 @@
...
 ```
 
-Ionide automatically detects the compiler and tools, but if it doesn't for some reason (for example, the Visual F# Tools were installed to a different directory), you can manually add the containing folder (`...\Microsoft SDKs\F#\4.0`) to your PATH.
+You must also install the [.NET Core SDK](https://www.microsoft.com/net/download/macos).
 
-### macOS
+### [Linux](#tab/linux)
 
-On macOS, Ionide uses [Mono](https://www.mono-project.com).  The easiest way to install Mono on macOS is via Homebrew.  Simply type the following into your terminal:
+On Linux, Ionide also uses [Mono](http://www.mono-project.com).  If you're on Debian or Ubuntu, you can use the following:
```
Comment (mairaw) [inline docs/fsharp/get-started/get-started-vscode.md]: nit: remove extra spaces between sentences (it happens frequently)

also, for this link the https link works for me

the following what? commands?

File: docs/fsharp/get-started/get-started-vscode.md [hunk 26]
```diff
@@ -134,8 +105,8 @@ If you followed the previous step steps, you should get the Visual Studio Code W
 This template generates a few things you'll find useful:
 
 1. The F# project itself, underneath the `ClassLibraryDemo` folder.
-2. The correct directory structure for adding packages via [`Paket`](https://fsprojects.github.io/Paket/).
-3. A cross-platform build script with [`FAKE`](https://fake.build/).
+2. The correct directory structure for adding packages via [`Paket`](http://fsprojects.github.io/Paket/).
+3. A cross-platform build script with [`FAKE`](http://fsharp.github.io/FAKE/).
```
Comment (mairaw) [inline docs/fsharp/get-started/get-started-vscode.md]: i'd bring the link that I fixed back

File: docs/fsharp/get-started/get-started-vscode.md [hunk 27]
```diff
@@ -134,8 +105,8 @@ If you followed the previous step steps, you should get the Visual Studio Code W
 This template generates a few things you'll find useful:
 
 1. The F# project itself, underneath the `ClassLibraryDemo` folder.
-2. The correct directory structure for adding packages via [`Paket`](https://fsprojects.github.io/Paket/).
-3. A cross-platform build script with [`FAKE`](https://fake.build/).
+2. The correct directory structure for adding packages via [`Paket`](http://fsprojects.github.io/Paket/).
```
Comment (mairaw) [inline docs/fsharp/get-started/get-started-vscode.md]: https link

File: docs/fsharp/index.md [hunk 28]
```diff
@@ -14,45 +13,69 @@ ms.assetid: ea27fb37-dad1-4bd4-a3cc-4f5c70767ae9
...
+F# is about productivity at its heart. The tooling support for F# is ubiquitous and full of advanced features.
 
-## The F# Software Foundation
+## Learning F# #
 
-Although Microsoft is the primary developer of the F# language and its tooling, F# is also backed by an independent foundation, the F# Software Foundation (FSSF).
+[Tour of F#](tour.md) gives an overview of major language features with lots of code samples. This is recommended if you are new to F# and want to get a feel for how the language works.
 
-The mission of the F# Software Foundation is to promote, protect, and advance the F# programming language, and to support and facilitate the growth of a diverse and international community of F# programmers.
+[Get started with F# in Visual Studio](get-started/get-started-visual-studio.md) if you're on Windows and want the full Visual Studio IDE experience.
```
Comment (mairaw) [inline docs/fsharp/index.md]: define what IDE is on first usage

File: docs/fsharp/index.md [hunk 29]
```diff
@@ -14,45 +13,69 @@ ms.assetid: ea27fb37-dad1-4bd4-a3cc-4f5c70767ae9
...
+
+[Get started with F# in Visual Studio for Mac](get-started/get-started-with-visual-studio-for-mac.md) if you're on macOS and want to use a Visual Studio IDE.
+
+[Get Started with F# in Visual Studio Code](get-started/get-started-vscode.md) if you want a lightweight, cross-platform, and feature-packed IDE experience.
+
+[Get started with F# with the .NET Core CLI](get-started/get-started-command-line.md) if you just want a great set of command-line tools.
```
Comment (mairaw) [inline docs/fsharp/index.md]: feels a bit marketing to me.

**Review by mairaw — APPROVED** [review]
I'm approving this but I think you should address the comments that Bill and I left before merging.

File: docs/fsharp/index.md (hunk 22 above)
Comment (cartermp) [reply docs/fsharp/index.md]: Perhaps, but most of the data I've seen about FP indicates that it's not something new programmers get into. Not until it's taught in beginner courses in university, at least.

**Comment by cartermp** [comment]
And now, to fix up the VSCode article and tour and numerous other backlog issues 😄

File: docs/fsharp/index.md (hunk 22 above)
Comment (BillWagner) [reply docs/fsharp/index.md]: That makes sense. I think the important point is to identify the audience so they can self-select the right locationn.
